uber/config.py

```diff
@@ -376,11 +376,11 @@ def _is_intstr(s):
         _interpolated = OrderedDict()
         for _desc, _val in _section.items():
             if _is_intstr(_val):
-                key = int(_val)
+                _price = int(_val)
             else:
-                key = getattr(c, _val.upper())
+                _price = getattr(c, _val.upper())
 
-            _interpolated[key] = _desc
+            _interpolated[_desc] = _price
```

Comment:
This was the heart of the duplicate price bug (everything else is just accounting for this change): because we were putting items into a dictionary keyed off the price, an item with the same price would clear out the previous entry.  Now we're keying off of the name, so that's no longer an issue.